source/_posts/2017/2017-01-20-4-emotional-reasons-I-quit-my-twitter.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: "4 Emotional and Honest Reasons I quit my Twitter"
```

Comment:
I think its phrase "x reasons to stop ...", "x reasons I did sth".

Well, I will try that :)